dictionary.txt

```diff
@@ -1091,12 +1091,31 @@ Demon queen
 Non-binary
 UwU
 OwO
-Genderfae
+genderfae
 **Want kiss gorls, am lesbean
-Bi
-Plural
-Prog-rock
-Encrypted
-Pizza
-Kissable
-Huggabl
+pan
+plural
+prog-rock
+encrypted
+pizza
+kissable
+huggable
+meow
+kitten
+bunny
+wife
+wifeing
+queer
+gay
+evil
+**girls,,,,
+**smooching girls
+femboy
+hot take
```

Comment:
On my personal thoughts, i'd prefer to avoid this one since "hot takes" are often related to bad stuff ?